python/tests/unit/updater/test_factory.py

```diff
@@ -0,0 +1,31 @@
+"""Test updater factory."""
+import mock
+
+from gilded_rose import dtypes
+from gilded_rose.updater import constants, factory
+
+
+def test_not_found():
+    """Test that when item has no updater."""
```

Comment:
Nit: phrase doesn't convey useful info, probably could be better with something like `"""Test item with no updater."""`

Reply:
https://github.com/benjamincerigo/GildedRose-Refactoring-Kata/pull/1/commits/1d237745230a535dd70454d63a818ccfa65cb6e4